src/main/java/com/marksem/crm/contract/DocumentImp.java

```diff
@@ -0,0 +1,8 @@
+package com.marksem.crm.contract;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+public interface DocumentImp<EntReq, EntRes> {
```

Comment:
We don't need interfaces for services in this project since we will always have only one implementation